app/main.py

```diff
@@ -1,3 +1,51 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, int):
+            return Distance(self.km + other)
+        return Distance(self.km + other.km)
```

Comment:
what if `other` will be not of the type `int`, `float`, and `Distance`? better check that `other` is `distance` object. Consider this in all methods